### ISSUE TITLE
When there's more than one richtext we have duplicated `id` attributes in the DOM

### ISSUE THREAD

**Issue by oiorain** [strapi/strapi#16382]
### Required System information

- Node.js version: 18
- NPM version: 9.5
- Strapi version: strapi-react18 
- Database: sqlite
- Operating system: macos
- Is your project Javascript or Typescript: js

### Steps to reproduce the behavior

1. created a content type with a dynamic zone
2. inserted a first element
3. when inserting the second element, I got a warning saying there was 2 elements with the same id.

### Screenshots

![Screenshot 2023-04-13 at 17 06 37](https://user-images.githubusercontent.com/4447713/231803087-ebfca5be-1346-4825-af9f-f2f5ebce9788.png)

**Comment by madhurisandbhor** [comment]
Could not reproduce

**Comment by gu-stav** [comment]
@madhurisandbhor I believe this is a valid issue and I have seen these warnings too in the past. There is even an attached PR with a fix: https://github.com/strapi/strapi/pull/16822

**Comment by oiorain** [comment]
We tried to reproduce it yesterday together with @madhurisandbhor  but we werent able to find the way again. My explanations are not that clear in retrospect. 
Do you remember where you saw it and how to trigger it?

**Comment by gu-stav** [comment]
@oiorain Just looking at the PR: the issue would arise I think as soon as more than one richtext editor is used in one edit view, because then the hardcoded ids are duplicated.

To see the warnings the "warnings" tabs in e.g. the chrome console must be enabled :)

Does that help?

**Comment by madhurisandbhor** [comment]
> @oiorain Just looking at the PR: the issue would arise I think as soon as more than one richtext editor is used in one edit view, because then the hardcoded ids are duplicated.
> 
> To see the warnings the "warnings" tabs in e.g. the chrome console must be enabled :)
> 
> Does that help?

Yes I have warnings enabled in chrome still these warnings are not visible, may be its a chrome version issue,
https://stackoverflow.com/questions/58027666/why-isnt-non-unique-id-warning-visible-in-google-chrome

I will take care of this PR https://github.com/strapi/strapi/pull/16822.

**Comment by gu-stav** [comment]
🤔 you are right Chrome doesn't display the warning any longer. The issue is real though and removing the IDs is a good improvement :)
